src/views/messages-view.jsx

```diff
@@ -879,63 +879,68 @@ class MessagesView extends React.Component {
 
           }, this);
         }
-        this.convertIntoThumbnails(ents, 0, (success) => {
-          if (success) {
-            const msg = Drafty.createQuote(header, cont, letterTileColorId(thisFrom));
-            this.setState({reply: {content: msg, seq: m.seq}});
-          }
-        });
 
-        return;
-      }
-    }
-  }
+        // Turn all images into thumbnails.
+        const promises = [];
+        ents.map((ex) => {
+          let p = new Promise((resolve, reject) => {
+            const scale = (origBlob) => {
+              imageScaled(origBlob, IMAGE_THUMBNAIL_DIM, IMAGE_THUMBNAIL_DIM, -1, false,
```

Comment:
Maybe make them square? It would look nicer and we would not need to care about corner case such as very tall or very wide images?